samples/tmo_shell/src/tmo_pm_sys.c

```diff
@@ -16,83 +16,92 @@
 #include <zephyr/pm/device.h>
 #endif
 
-int cmd_pmsysactive(const struct shell *shell, int argc, char** argv)
+const struct device *tmo_dev_list[] = {DEVICE_DT_GET(DT_NODELABEL(rs9116w)),
+				       DEVICE_DT_GET(DT_NODELABEL(sonycxd5605)),
+				       DEVICE_DT_GET(DT_NODELABEL(murata_1sc))};
+
+#if CONFIG_PM_DEVICE
+static void tmo_pm_sys_lp_wrapper(bool turn_off)
+{
+	const struct device *dev = NULL;
+	for (int i = 0; i < sizeof(tmo_dev_list) / sizeof(tmo_dev_list[0]); i++) {
+		dev = tmo_dev_list[i];
+		pm_device_action_run(dev, PM_DEVICE_ACTION_SUSPEND);
+		if (turn_off && dev != DEVICE_DT_GET(DT_NODELABEL(murata_1sc))) {
+			pm_device_action_run(dev, PM_DEVICE_ACTION_TURN_OFF);
+		}
+	}
+	dev = PWMLEDS;
+	led_off(dev, 0);
+	led_off(dev, 1);
+	led_off(dev, 2);
+	led_off(dev, 3);
```

Comment:
Rather than the hard-coded values, let's include the board.h file and use the macros defined therein.